src/apps/watchfaces/OswAppWatchfaceMonotimer.cpp

```diff
@@ -0,0 +1,146 @@
+
+#include "./apps/watchfaces/OswAppWatchfaceMonotimer.h"
+// #define GIF_BG
+
+#ifdef ANIMATION
+#include <anim_matrix.h>
+#endif
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+
+#ifdef GIF_BG
+#include "./apps/_experiments/gif_player.h"
+#endif
+
+void OswAppWatchfaceMonotimer::drawNShiftedTicks(Graphics2D *gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t color) {
```

Comment:
I think it would be good to write an explanation about the factors that go into the function.
```
/*
@param sample blah blah
*/
void OswAppWatchfaceMonotimer::drawNShiftedTicks()...
```

Reply:
What kind of functions does this function?

Reply:
See https://github.com/Open-Smartwatch/open-smartwatch-os/pull/288/files/745feaa6610539a2137ba35bee33573faa40b935#r962147375